pkg/crc/preset/preset.go

```diff
@@ -22,6 +22,11 @@ var presetMap = map[Preset]string{
 	Microshift: string(Microshift),
 }
 
+const (
+	PodmanDeprecatedWarning = "The Podman preset is deprecated and will be removed in a future release. Consider" +
+		" rather using a Podman Machine managed by Podman Desktop: https://podman-desktop.io"
```

Comment:
👍, @themr0c will appreciate this.

Reply:
thanks, I hope so as I took the words from him 😄

Reply:
/me knows. I approved those ;-)